src/components/CartProvider.jsx

```diff
@@ -0,0 +1,34 @@
+import { createContext, useState } from "react";
+
+export const CartContext = createContext();
+
+export default function CartProvider({ children }) {
+    const key = "cart";
+
+    const [cart, setCart] = useState(console.log(localStorage[key]) || [])
```

Comment:
```suggestion
    const [cart, setCart] = useState(console.log(localStorage[key]) || [])
```
I don't think console.log belongs here